content/en/docs/tasks/administer-cluster/memory-manager.md

```diff
@@ -178,7 +179,7 @@ Example usage:
 
 or
 
-`--reserved-memory 0:memory=1Gi --reserved-memory 1:memory=2Gi`
```

Comment:
in general both formats should work, so please add `or` example instead of updating the old one